src/plugins/intel_cpu/src/nodes/subgraph.cpp

```diff
@@ -535,6 +538,13 @@ bool Snippet::created() const {
 void Snippet::generate(const jit_snippets_compile_args* jcp) {
     ov::pass::Manager pre_dialect;
     pre_dialect.register_pass<ConvertToSwishCPU>();
+    if (context->getConfig().enforceBF16) {
```

Comment:
Also these passes should be registered only in cases when subgraph contains `matmul` at least.
So let's create the method has_domain_sensetive_ops() that [returns true if Subgraph has](https://github.com/openvinotoolkit/openvino/blob/dec425c4085f7b95ed564d591295a40ebb55476c/src/common/snippets/src/op/subgraph.cpp#L65) and add condition with this method here to register needed passes.

Reply:
Why not in this PR? What is the problem? It's just needed to avoid useless passes for `Subgraph`s with only `Eltwise`. For what we should call `MatMulToBrgemm()` and `EnforcePrecision()`, if we don't have `MatMu`l in `Subgraph`?

Reply:
done, thanks